config/settings/base.py

```diff
@@ -23,7 +23,8 @@
     for name_and_email in os.getenv("CORGI_ADMINS", ";root@localhost").split(",")
 )
 
-DOCS_URL = os.getenv("CORGI_DOCS_URL")
+# Default to public Jira project if we can't point at any ops docs.
+DOCS_URL = os.getenv("CORGI_DOCS_URL") or "https://issues.redhat.com/projects/SECDATA"
```

Comment:
.getenv() defaults to None and supports a second arg for a custom default, just like .get(). So this can just be:
```suggestion
DOCS_URL = os.getenv("CORGI_DOCS_URL", "https://issues.redhat.com/projects/SECDATA")
```